backend/src/main/java/com/redhat/cloud/notifications/db/repositories/BehaviorGroupRepository.java

```diff
@@ -537,6 +558,27 @@ public void deleteBehaviorGroupFromEventType(final UUID eventTypeUuid, final UUI
         }
     }
 
+    @Transactional
+    public void deleteEndpointFromBehaviorGroup(final UUID behaviorGroupUuid, final UUID endpointUuid, final String orgId) {
+        final var deleteFromEventTypeQuery =
+            "DELETE FROM BehaviorGroupAction AS bga " +
```

Comment:
We should check (here or in an upper layer) that `endpointUuid` is owned by the current `orgId`.

Reply:
This check is akready done by a sub method called by an upper layer who will throw and exception, but it's not obious from a mantenance perspective. You are right, lets check it here to avoid confusion.